src/index.test.ts

```diff
@@ -76,7 +74,7 @@ test('Test placeholder', async (t) => {
     )
     .resolves(fakePoolResult());
 
-  const store = await newEventStore(getFakePool(readStub), emit);
+  const store = await newEventStore(getFakePool(readStub), fakeEmitter);
```

Comment:
There's another place you'll need to change emit to fakeEmitter so it'll pass the CI